.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```

Comment:
was this accidentally added? you may want to add ".project" to ".gitignore" instead

Reply:
Removed `.project` and added `.gitignore` line